core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -365,114 +397,88 @@ struct MDRangePolicy : public Kokkos::Impl::PolicyTraits<Properties...> {
   bool impl_tune_tile_size() const { return m_tune_tile_size; }
 
  private:
-  void init() {
-    // Host
-    if (true
+  // Host
+  template <typename ExecutionSpace>
+  void init(const ExecutionSpace&) {
+    const index_type max_threads       = std::numeric_limits<int>::max();
+    const index_type max_tile_size     = 0;
+    const index_type default_tile_size = 2;
+    init_helper(max_threads, max_tile_size, default_tile_size, max_threads);
+  }
+
 #if defined(KOKKOS_ENABLE_CUDA)
-        && !std::is_same<typename traits::execution_space, Kokkos::Cuda>::value
+  void init(const Kokkos::Cuda& space) {
+    const index_type max_threads =
+        space.impl_internal_space_instance()->m_maxThreadsPerSM;
+    const index_type max_tile_size       = 16;
+    const index_type default_tile_size   = 2;
```

Comment:
what is default tile size?

Reply:
The tile size for the dimensions that are not the largest one.